lib/tasks/tasks.rake

```diff
@@ -37,9 +37,11 @@ namespace :crucible do
     puts "Test Scripts generated in #{b.real} seconds."
   end
 
-  def execute_test(client, test)
-    output_results Crucible::Tests::Executor.new(client).execute(test)
+  def execute_test(client, key)
+    executor = Crucible::Tests::Executor.new(client)
+    output_results executor.execute(executor.find_test(key))
   end
+
   def execute_multiserver_test(client, client2, test)
     output_results Crucible::Tests::Executor.new(client, client2).execute(test)
```

Comment:
If you call execute_multiserver_test, do you have the test already, or are you really passing in the key (like in the regular execute_test)?